pkgs/top-level/all-packages.nix

```diff
@@ -32521,6 +32521,8 @@ with pkgs;
 
   meerk40t-camera = callPackage ../applications/misc/meerk40t/camera.nix { };
 
+  mmj2 = callPackage ../applications/editors/mmj2 {};
```

Comment:
This should be removed and the package should then be moved to the `pkgs/by-name/so/some-example/package.nix` hierarchy (: